src/provider/provider_file_memory.c

```diff
@@ -41,6 +42,9 @@ typedef struct file_memory_provider_t {
     unsigned visibility; // memory visibility mode
     size_t page_size;    // minimum page size
 
+    /// IPC is enabled only for UMF_MEM_MAP_SHARED or UMF_MEM_MAP_SYNC visibility
```

Comment:
why using `///` instead of `//` (this is not public API, it will not show up on the web docs)

Reply:
Just a mistake.
Done.